assets/js/util.js

```diff
@@ -336,8 +336,8 @@ function handleAPyRequestCompletion(requestDuration) {
     var cumulativeAPyRequestDuration;
 
     if(lastNAPyRequestDurations.length === INSTALLATION_NOTIFICATION_REQUESTS_BUFFER_LENGTH) {
-        cumulativeAPyRequestDuration = lastNAPyRequestDurations.reduce(function (sum, value) {
-            return sum + value;
+        cumulativeAPyRequestDuration = lastNAPyRequestDurations.reduce(function (total, currentValue) {
```

Comment:
This sidesteps the problem rather than actually addressing it. Tricking the linter isn't the proper course of action :) Something like `duration` that's more specific makes more sense than `value`.